help/en/releasenotes/current-draft-note.shtml

```diff
@@ -412,6 +412,12 @@
           <i>Paste copy</i>. With it, you can paste from the clipboard as many times
           as needed.</li>
         </ul>
+        <ul>
```

Comment:
Don't start a new unordered list.

Reply:
I fixed the <ul>, but the difference between the buttons when disabled appears to be a Mac vs Windows thing:  this is how it appears to me:

![image](https://user-images.githubusercontent.com/6137710/197594445-40a66dbf-54d4-4401-9091-12d98956a190.png)

I selected some text and can copy it but can't change it. -- How do we make the buttons different on a Mac?

Reply:
Since a button implies the possibility of an action, a label is better for showing unchangeable data, such as the Table type label.

The ability to copy the file name is fine.

Reply:
Replace this:
```
    <h3>LogixNG</h3>
        <a id="LogixNG" name="LogixNG"></a>
        <ul>
          <li>Fixes a serious bug when deleting a LogixNG. Not all ConditionalNGs of
          that LogixNG was deleted but that's fixed now.</li>
          <li>Update the Global Variable table browser to provide detail information.</li>
          <li>LogixNG Formula now supports the boolean XOR operator <i>^^</i>.</li>
          <li>The ConditionalNG editor and the Module editor now have the option
          <i>Paste copy</i>. With it, you can paste from the clipboard as many times
          as needed.</li>
        </ul>
        <ul>
          <li>Adds support of multiple types of CSV files to LogixNG Tables
          Tab separated files are the default, but you may choose Comma
          as the separator. This allows you load CSV files created by
          spreadsheet programs.</li>
        </ul>
```

with:

```
    <h3>LogixNG</h3>
        <a id="LogixNG" name="LogixNG"></a>
        <ul>
          <li>Fixes a serious bug when deleting a LogixNG. Not all ConditionalNGs of
          that LogixNG was deleted but that's fixed now.</li>
          <li>Update the Global Variable table browser to provide detail information.</li>
          <li>LogixNG Formula now supports the boolean XOR operator <i>^^</i>.</li>
          <li>The ConditionalNG editor and the Module editor now have the option
          <i>Paste copy</i>. With it, you can paste from the clipboard as many times
          as needed.</li>
          <li>Adds support of multiple types of CSV files to LogixNG Tables.
          Tab separated files are the default, but you may choose Comma
          as the separator. This allows you load CSV files created by
          spreadsheet programs.</li>
        </ul>
```